src/main.cpp

```diff
@@ -2356,6 +2356,12 @@ bool ProcessBlock(CNode* pfrom, CBlock* pblock)
 
     LogPrintf("ProcessBlock: ACCEPTED\n");
 
+	// If turned on stakeforcharity, send a portion of stake reward to savings account address
```

Comment:
should we put this into the "soft fork check" for the stability fixes, so that it only starts sending us parts of their stakes AFTER the RoI/Maturation changes are in?